fastlane/Fastfile

```diff
@@ -5,8 +5,14 @@ default_platform(:ios)
 platform :ios do
   desc 'Builds the project and runs tests'
   lane :test do
+    xcodebuild(
+      scheme: 'WordPressShared',
+      xcargs: '-resolvePackageDependencies'
+    )
```

Comment:
~~Interesting... I assume the tests run didn't work without this.~~

~~Have you tried passing the `xcargs` option to `run_tests`?~~

~~The action has parameters such as `disable_package_automatic_updates` and `skip_package_dependencies_resolution` ([both defaulting to `false`](https://docs.fastlane.tools/actions/scan/)) which makes me think it should be able to handle this without a dedicated step.~~

Okay. Silly me, I should have done my reading before commenting 😓 

I can see the error this addresses from the CI builds, e.g. [this one](https://buildkite.com/automattic/wordpress-shared-ios/builds/97#018479fc-7df6-4fab-8f1a-491effdc5525/461-526).

```
[...] Cannot open "Documentation.docc" as a "Swift Package Folder" because it is already open as a "Folder"." 
[...]
[06:35:08]: ▸ ❌  error: Build service could not create build operation:
  internal error:
    could not generate PIF because the workspace has not finished loading or is still waiting for package resolution
```

Seems like a tooling bug to me. Worth adding a note in the code about it?

Reply:
> Seems like a tooling bug to me 

I noticed CI for this project is on Xcode 14.0. I opened #323 to see if Xcode 14.1 solved the alleged tooling issue.

Reply:
I have seen similar error in Xcode 13 before, my guess is it's caused by some race condition within Xcode between creating the Swift Package's Xcode project and loading the project. But yeah, hopefully this is no longer an issue in Xcode 14.1.

Another thing is, if this issue continues to show up, it would be a good idea to move this additional command into bash-cache's `validate_swift_package` script. Also, it's probably also worth to make the script self contained, rather than relying on the projects to have a `test` lane.